### DISCUSSION TITLE
Personal Account to Organisation conversion error

### DISCUSSION THREAD

**Discussion by adrianvdh** [community/community#67463] [New to GitHub] [answered]
Hi there

I registered a personal account named Serveox.

I then proceeded to convert this account into a Github Organisation. I saw all the warnings and accepted the terms.

The process started, and then I was instantly signed out. No emails were sent to me thereafter. I cannot seem to access this Organisation, and I'm not a member of this org either. How can I regain access to it??

Please will you assist me?

https://github.com/Serveox

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 , 

When you log in to your personal account and navigate to your settings (click your profile photo) then `Your Organizations`, it isn't listed under your organizations? 

![GitHub Account Settings Menu](https://github.com/community/community/assets/92824518/3784cda4-2775-4857-854f-c840f0f63508)

**Reply by adrianvdh** [reply]
Hi there @queenofcorgis 

Thank you for replying :)

Currently no it's not listed

![Screenshot 2023-09-19 at 17 01 53](https://github.com/community/community/assets/25223854/fb123d29-da9b-4c27-b565-36039eac4947)

**Reply by queenofcorgis** [reply]
@Adrianvdh ah I figured that was the case - this is probably more account-specific then so I'm going to recommend you open a ticket with our lovely [support team](https://support.github.com/). Best of luck!

**Reply by adrianvdh** [reply]
@queenofcorgis, thank you.

I've already opened up a support ticket last week, however, I haven't received any response.

Is there anything you can do?

**Reply by queenofcorgis** [reply]
Thanks for your patience. I am not able to escalate it, but I checked on your ticket and rest assured it is in the right place. Response time depends on the amount of tickets ahead of yours and our team's resources.

**Comment by swiftblr** [comment]
@Adrianvdh were you able to find solution to this ? I am also facing same issue.

**Reply by adrianvdh** [reply, marked as answer]
Hi @swiftblr the problem I actually had was that I chose the incorrect organisation owner.

I have a personal GitHub account and an ex-employer GitHub account with the same profile pictures. So I chose the ex-employer Github account which I no longer have access to.

To resolve this issue, I emailed GitHub support using the Serveox GitHub organization's email address, asking that they add my personal GitHub account as an organization owner.

Support kindly did this after a couple of days and I could then regain access to the organisation and remove the original owner.

Let me know if you need me to clarify any further.

**Reply by Terencesquared** [reply]
Whats the email for Serveox?

**Comment by swiftblr** [comment]
thanks Adrin. this is useful information.

On Tue, 24 Oct 2023 at 15:12, Adrian van den Houten <
***@***.***> wrote:

> Hi @swiftblr <https://github.com/swiftblr> the problem I actually had was
> that I chose the incorrect organisation owner.
>
> I have a personal GitHub account and an ex-employer GitHub account with
> the same profile pictures. So I chose the ex-employer Github account which
> I no longer have access to.
>
> To resolve this issue, I emailed GitHub support using the Serveox GitHub
> organization's email address, asking that they add my personal GitHub
> account as an organization owner.
>
> Support kindly did this after a couple of days and I could then regain
> access to the organisation and remove the original owner.
>
> Let me know if you need me to clarify any further.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/67463#discussioncomment-7368254>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBYOK32TKWJJ3DXLVV67DMLYA6EPXAVCNFSM6AAAAAA44MS566VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGNRYGI2TI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Thanks,
Nayyan Mujadiya
